lib/collection/request-body.js

```diff
@@ -17,26 +17,26 @@ _.inherit((
      * @constructor
      * @extends {PropertyBase}
      *
-     * @param {Object} options
+     * @param {*} options
      */
     RequestBody = function PostmanRequestBody (options) {
```

Comment:
This should work like headers in some way. If `options` is specifically string then we modify options to:

```
options = {
    mode: 'raw',
    raw: options
}

Reply:
Yes, that is the current behaviour.